### PR TITLE
Improvements to "Commonly used parameters for transpilation"

### PR DESCRIPTION
**Pull request by kevinsung**
Part of https://github.com/Qiskit/documentation/issues/885

- Don't use `transpile` function
- Rewrite the "Initial layout" section to add code examples and correct information about what arguments are allowed.

### CODE REVIEW

File: docs/transpile/common-parameters.ipynb
Comment (javabster) [inline docs/transpile/common-parameters.ipynb:1]: for the "Random number generator seed" title I think we should keep the original "seed transpiler" wording so it's consistent with the other titles (i.e. the titles match the actual name of the argument they are describing)

Comment (javabster) [inline docs/transpile/common-parameters.ipynb:1]: "In the example below, we construct an initial layout for the FakeSherbrooke mock backend by creating a [Layout](/api/qiskit/qiskit.transpiler.Layout) object"

`FakeSherbrooke` not FakeSherbrooke

Comment (javabster) [reply docs/transpile/common-parameters.ipynb:1]: "You can visit https://quantum.ibm.com/services/resources to access device graph diagrams with physical qubits labeled."

I think there may also be a function in qiskit that lets you do this? Might be a nice addition

**Review by javabster — COMMENTED** [review]
Overall liking the improvements to this piece, just left a couple of minor comments :)

File: docs/transpile/common-parameters.ipynb
Comment (kevinsung) [reply docs/transpile/common-parameters.ipynb:1]: I think the titles should either be
1. The name of the concept (random number generator seed)
2. The exact name of the argument (`seed_transpiler`)

What do you think? If you want to go with (2) then I'll change the other titles to match.

Comment (kevinsung) [reply docs/transpile/common-parameters.ipynb:1]: > `FakeSherbrooke` not FakeSherbrooke

Done.

> I think there may also be a function in qiskit that lets you do this? Might be a nice addition

For 127 qubits the output has overlapping text and is pretty ugly (see image below). I don't think we should include it. I could just refer them to the function though. What do you think?

image

Comment (javabster) [reply docs/transpile/common-parameters.ipynb:1]: hmm I prefer 2. but I know that we've got a css bug with code blocks in headers so maybe we should stick with 1 then if 2 looks terrible 🤔

Comment (javabster) [reply docs/transpile/common-parameters.ipynb:1]: is the overlapping text fixed if you make the matplotlib axis bigger?

Comment (kevinsung) [reply docs/transpile/common-parameters.ipynb:1]: Yes, thanks for the suggestion. Done!

Comment (kevinsung) [reply docs/transpile/common-parameters.ipynb:1]: As we discussed in person, I went with (1).

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**